website/docs/faqs/Docs/documenting-macros.md

```diff
@@ -27,3 +27,26 @@ macros:
 ```
 
 </File>
+
+## Document a custom materialization
+
+When you create a [custom materialization](/guides/create-new-materializations), dbt creates an associated macro with the following format:
+```
+materialization_{materialization_name}_{adapter}
+```
+
+To document a custom materialization, use the format above to determine the associated macro name(s) to document.
```

Comment:
removig the word 'above' for accessibility. spatial descriptions like "above" can be confusing for people who rely on screen readers or have difficulty understanding spatial relationships. 

```suggestion
To document a custom materialization, use the previously mentioned format to determine the associated macro name(s) to document.
```